### DISCUSSION TITLE
What is the correct config set-up for a Nuxt and Flask monorepo using 'flask-restx'? Is zero-config possible?

### DISCUSSION THREAD

**Discussion by bigsee** [vercel/vercel#4731] [Help]
Hello!

I have a project with the client built using Nuxt. Everything works fine.

I would like to include some services built in Python with Flask. I have looked at various discussions, most notably (issue [#163](https://github.com/vercel/now-examples/issues/163) in now-examples).

I have been able to get the simple hello world running with zero-config using [the official Python runtime with WSGI support](https://vercel.com/docs/runtimes#advanced-usage/advanced-python-usage). However, I don't have all the benefits of the Flask's REST extension(s).

I have also been able to get a simple 'Hello world' working with flask-restx on Vercel (like the one provided earlier in the issue above). Here is a link to that project: https://vercel.com/bigsee/vercel-flask-restx/2qv8fuill/source

My questions are:

1. Is it possible to get something like [flask-restx](https://flask-restx.readthedocs.io/en/latest/quickstart.html) working using a zero-config set up?
2. If not, what is the correct way to approach deploying a monorepo where the frontend needs no config but the backend needs a custom build specified in a vercel.json file?

Thank you in advance!

**Comment by paulogdm** [comment]
Hello,

We do not recommend that you deploy Flask to our platform. Serverless Functions should work as helpers to your frontend. Complex API setups should be hosted in better-suited options.

Officially, we only support Nuxt.js when doing SSG.

**Comment by bigsee** [comment]
Hello @paulogdm

Thank you for your feedback. I guess what I'm struggling with is understanding at what point an API setup becomes 'complex'? I'm not trying to do anything particularly special - just a few routes to fetch data. And I'm just using Flask to abstract away things that it can handle out of the box.

It was my understanding that [WSGI support was recently added to your official Python runtimes](https://vercel.com/docs/runtimes#advanced-usage/advanced-python-usage) precisely to enable using frameworks like Flask. Is the plan then to reverse this at some point?

I'm just a bit confused as to when I should use Vercel and when I shouldn't.

**Comment by bigsee** [comment]
P.S. I realise a year is a long time in tech, but it read like the plan was very much to support Flask in [this important Github discussion](https://github.com/vercel/now-examples/issues/163).
